Makefile

```diff
@@ -251,7 +251,7 @@ prepare-and-launch-docker-to-run-tests-within: docker-test-build launch-docker-t
 .PHONY: test-checkstyle-standalone
 test-checkstyle-standalone: test-shellcheck test-yaml test-critic test-js-style
 ifeq ($(CONTAINER_TEST),1)
-test-checkstyle-standalone: test-check-containers
+test-checkstyle-standalone: test-check-containers test-containers-compose
```

Comment:
still wrong. Again, `test-containers-compose` is not a "style check"

Reply:
May I recommend, on a general note, that you try and suggest what the correct place is or how to identify it? Instead of saying "wrong" repeatedly.

Reply:
https://github.com/os-autoinst/openQA/pull/3773#discussion_r592213208

Reply:
> not as part of "checkstyle" but as part of the dynamic tests, e.g. above in line 131+133

I guess that refers to the first two variants of the `test:` target 🤔 Should both be guarded by `ifeq ($(CONTAINER_TEST),1)`?

Reply:
I don't know exactly how it should look like but it should be clear that we want to run the functional, dynamic container tests as part of complete test runs as well as in exactly one circleCI test job while not calling it a "style check". If you want a complete description in code how it would look like I would be better off doing the change myself which I don't want to do unless you ask me because previously there were also complains when I did something like this.

Reply:
```make
.PHONY: test        
ifeq ($(TRAVIS),true)
test: run-tests-within-container
else
ifeq ($(CHECKSTYLE),0)
test: test-with-database
else                               
test: test-checkstyle-standalone test-with-database
endif
ifeq ($(CONTAINER_TEST),1)
test: test-containers-compose
endif
endif
```

I **think** this is what we want, i.e. add the new target to `test` if `CONTAINER_TEST` is set, but outside the `TRAVIS` guard - not tested, though

Reply:
Thanks, changed

Reply:
Not resolved though. As long as test-containers-compose is called as part of style checks I will still consider this wrong

Reply:
> I don't know exactly how it should look like but it should be clear that we want to run the functional, dynamic container tests as part of complete test runs as well as in exactly one circleCI test job while not calling it a "style check". If you want a complete description in code how it would look like I would be better off doing the change myself which I don't want to do unless you ask me because previously there were also complains when I did something like this.

Please, go ahead

Reply:
Just remove this one-line change here and it's ok